ynr/apps/popolo/api/next/filters.py

```diff
@@ -0,0 +1,42 @@
+from django_filters import filterset, filters
+
+from popolo.models import Organization, Post, Membership
+
+
+class OrganizationFilterMixin(filterset.FilterSet):
+    class Meta:
+        model = Organization
+        fields = ["last_updated"]
+
+        last_updated = filters.DateTimeFilter(
```

Comment:
This is an example pf what I mentioned earlier about with the indentation - `last_updated` is indented too far right so that it ends up being defined on the `Meta` class when it needs to be defined on the `OrganizationFilterMixin` class. So you would want to change to:
```
class OrganizationFilterMixin(filterset.FilterSet):

    last_updated = filters.DateTimeFilter(
         field_name="modified",
         lookup_expr="gt",
         label="Last updated",
         help_text="An ISO datetime",
     )
     
     class Meta:
         model = Organization
         fields = ["last_updated"]